mill-scalafix/src/com/goyeau/mill/scalafix/ScalafixModule.scala

```diff
@@ -63,6 +66,7 @@ object ScalafixModule {
     os.pwd
   )
 
+  @nowarn("msg=parameter scalaBinaryVersion in method fixAction is never used")
```

Comment:
I assume you left the parameter for binary compatibility in the patch line? Nitpicking: we could create an overload (easier to clear later) and get rid of the propagation of `scalafixScalaBinaryVersion` for the real code path.

Reply:
👍